bundles/tools.vitruv.change.propagation/src/tools/vitruv/change/propagation/impl/ChangePropagator.xtend

```diff
@@ -30,14 +30,34 @@ class ChangePropagator {
 	val ChangeRecordingModelRepository modelRepository
 	val ChangePropagationSpecificationProvider changePropagationProvider
 	val InternalUserInteractor userInteractor
-
+	var boolean isPropagatingTransitively = true
+	
+	/**
+	 * Creates a change propagator to which changes can be passed, which are
+	 * propagated using the given <code>changePropagationProvider</code> and using the given
+	 * <code>userInteractor</code>.
```

Comment:
double `using the given`

Reply:
Do you mean this is a stylistic issue, i.e., better write "using the given changePropagationProvider that provies change propagations calling the given userInteractor when required" or something like that?

Reply:
I only meant that the sentence could be simplified to "using the given <code>changePropagationProvider</code> and (the given) <code>userInteractor</code>"

Reply:
Fixed in d7fba5e